### DISCUSSION TITLE
Developer's Dashboard doesn't show any of my projects

### DISCUSSION THREAD

**Discussion by ForMiSoft** [tidal-music/discussions#106] [Q&A]
I wanted to delete one of my projects in the dashboard, so I opened the project settings for editing, but there wasn’t an option to delete it, only to disable it. I disabled it, and then a delete button appeared as an option. So I clicked it, but an error message came up saying I couldn’t delete a disabled project. I tried to enable the project, but every action resulted in errors, and since then I haven’t been able to see any of my projects. I can create new ones, but they don’t show up either.

![dashboard](https://github.com/user-attachments/assets/67b68875-b237-4875-878c-6d6d257676ac)

And when I try to reload the page it says:

![Screenshot_20241015_200254_Chrome~2.jpg](https://github.com/user-attachments/assets/600d32cf-8fb5-408c-82f9-db1737ce1b99)

**Comment by aliaksei-taliuk** [comment]
@ForMiSoft hi, is the problem still persists? I quickly checked and couldn't reproduce it.

**Reply by aliaksei-taliuk** [reply]
If the problem is still there could you please share your email you used for signing up for the Developer Portal so I can check logs.

**Reply by aliaksei-taliuk** [reply]
will keep you posted

**Reply by ForMiSoft** [reply]
I don't know if you did something in the database, but it's working again. Thank you.

**Reply by aliaksei-taliuk** [reply]
please don't do anything, I temporarily hided the broken app you had. I'll un-hide it soon. Will let you know when it's fixed

**Reply by ForMiSoft** [reply]
Can you delete it?

**Reply by aliaksei-taliuk** [reply]
Yes, I can delete if you don't mind. But first I'll try to fix the root cause that led to this situation. Feel free to try to create a new one.

**Reply by ForMiSoft** [reply]
I already created a new one yesterday, I don't need that broken project anymore.

**Reply by aliaksei-taliuk** [reply]
ok then, let me know if anything else pops up

**Reply by ForMiSoft** [reply]
The same again. It shouldn't be so hard to fix it, it's easily replicable, it occurs always after deleting a project.
